waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -607,22 +611,27 @@ proc addAll*(rlnInstance: RLN[Bn256], list: seq[IDCommitment]): bool =
 type RegistrationEventHandler  = proc(pubkey: Uint256, index: Uint256): void {.gcsafe, closure, raises: [Defect].}
 
 
-proc subscribeToGroupEvents(ethClientUri: string, contractAddress: Address, blockNumber: string = "0x0", handler: RegistrationEventHandler) {.async, gcsafe.} = 
+proc subscribeToGroupEvents(ethClientUri: string, ethAccountAddress: Address, contractAddress: Address, blockNumber: string = "0x0", handler: RegistrationEventHandler) {.async, gcsafe.} = 
   ## connects to the eth client whose URI is supplied as `ethClientUri`
   ## subscribes to the `MemberRegistered` event emitted from the `MembershipContract` which is available on the supplied `contractAddress`
   ## it collects all the events starting from the given `blockNumber`
   ## for every received event, it calls the `handler`
   
   # connect to the eth client
-  let web3 = await newWeb3(ETH_CLIENT)
+  let web3 = await newWeb3(ethClientUri)
   # prepare a contract sender to interact with it
   var contractObj = web3.contractSender(MembershipContract, contractAddress) 
+  web3.defaultAccount = ethAccountAddress 
+  #  set the gas price twice the suggested price in order for the fast mining
+  # let gasPrice = int(await web3.provider.eth_gasPrice()) * 2
 
   # subscribe to the MemberRegistered events
   # TODO can do similarly for deletion events, though it is not yet supported
   discard await contractObj.subscribe(MemberRegistered, %*{"fromBlock": blockNumber, "address": contractAddress}) do(pubkey: Uint256, index: Uint256){.raises: [Defect], gcsafe.}:
     try:
       debug "onRegister", pubkey = pubkey, index = index
+      echo "onRegister",  pubkey,  index
```

Comment:
Lingering `echo`? :)

Reply:
Not any more :) https://github.com/status-im/nwaku/pull/1023/commits/cbff025143fc1417c620a5a6374c61575f290630